examples/SolidColor.cpp

```diff
@@ -129,18 +113,15 @@ int main(int argc, char* argv[]) {
     // an HMD.  Do this using the OSVR RenderManager interface,
     // which maps to the nVidia or other vendor direct mode
     // to reduce the latency.
-    osvr::renderkit::RenderManager* render =
-        osvr::renderkit::createRenderManager(context.get(),
-        graphicsLibrary.c_str());
+    std::unique_ptr<osvr::renderkit::RenderManager> render(
```

Comment:
At least in some environments, objects in main() are not destroyed at program exit, so don't have their destructors called.  This includes std::unique_ptr().  I first encountered this on an ARM-based Ubuntu embedded controller board, but I hear that it also happens elsewhere.